### DISCUSSION TITLE
Should we just make our own compositor with CoMo?

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2578] [Q&A]
Should we just _make_ a compositor for ourselves? We could still be agnostic, but having a fallback might be a good idea. I've been looking at CoMo and Louvre recently, and wondering if either one would be suited for our plans; if we could make something that can be styled by .QSS stylesheets, so much the better.

**Comment by normal-max** [comment]
Window shadows are useful to visually separate overlapping windows and improve perception of content. Compositor is not just for decorations.

**Reply by stefonarch** [reply]
That is quite true, at least for stacking Wms.

![immagine](https://github.com/user-attachments/assets/2e56cfb4-f7f8-4195-ae3b-a600a4995e3c)

>Should we just make a compositor for ourselves? 

No, as it's an not easy task,  we've too less manpower and there are very good compositors already around - no need to reinvent the wheel.
